twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/QueryCompletedEventScriber.java

```diff
@@ -55,6 +61,23 @@ public void handle(QueryCompletedEvent event)
     }
   }
 
+  private static long getLongOrZero(String strVal)
+  {
+    try {
+      return DataSize.valueOf(strVal).toBytes();
+    }
+    catch (IllegalArgumentException e) {
+      log.warn(e, "Failed to parse io.airlift.units.DataSize string, returning 0");
+      return 0;
+    }
+  }
+
+  private static long getTotalDataTransferInBytes(JsonObject stage)
```

Comment:
Why sum all data transfer of all stages here, instead of persisting the transfer from each stage as a map? By rolling up here we lose the ability to review data transfer by stage which could be useful.